lib/router/index.js

```diff
@@ -76,11 +76,3 @@ export const createRouter = function (...args) {
 
 // Export the actual Router class, which is usually used inside the server
 export const Router = _Router
-
-export function _notifyBuildIdMismatch (nextRoute) {
```

Comment:
Now we can't do this.
If the app updated in between the SSR load and the JSON page load, app could break.

To fix this, we still send the BUILD_ID to the client and check is with the new one.

For this, we could generate a BUILD_ID based webpack hashes and yarn.lock, package.json etc.

@rauchg what do you think?